docs/configuration/overview.md

```diff
@@ -0,0 +1,25 @@
+---
+id: overview
+sidebar_position: 0
+sidebar_label: Configuration
+description: Learn how to configure Yazi.
+---
+
+# Configuration
+
+There are three configuration files for Yazi:
+
+- [`yazi.toml`](./yazi.md) - Configuration of file management-related functions
+- [`keymap.toml`](./keymap.md) - Configuration of keybindings
+- [`theme.toml`](./theme.md) - Yazi color scheme
+
+Yazi presets these configurations upon release, and uses them by default.
+
+If you want to use your own configurations and modify them as needed, please copy them from https://github.com/sxyazi/yazi/tree/main/yazi-config/preset to:
```

Comment:
```suggestion
We suggest copying these files from the [default configuration](https://github.com/sxyazi/yazi/tree/main/yazi-config/preset) to start, as the defaults are subject to change at any time and disrupt your experience.

The default configuration directories are:
```

Reply:
Why do we suggest users create their own configurations?

Reply:
Why else are they on the Configuration section? It's the same thing you are saying.

Reply:
I added the "If you wish to configure Yazi" line there. Seems clear enough to me.

Reply:
"as the defaults are subject to change at any time and disrupt your experience", this sentence is very strange.

What is the causal relationship between it and "copying these files to start"? Is it "because the default configuration files will change, so you need to copy the default configuration to begin"?

Reply:
Yes. To start as in like the base files to edit. Is that not what you are encouraging? Or do you want that just for keybinds?

Reply:
Re: my latest comment, do you want people to copy `yazi.toml` and `keymap.toml` from the preset to start configuring or just `keymap.toml`?

Reply:
I read the last sentence several times, but it still feels strange. Are you referring to the "defaults" in the Yazi release presets or the latest configuration files on GitHub?